packages/api/src/platforms/vtex/utils/enhanceSku.ts

```diff
@@ -1,8 +1,18 @@
 import type { Product, Item } from '../clients/search/types/ProductSearchResult'
+import { sanitizeHtml } from './sanitizeHtml'
 
 export type EnhancedSku = Item & { isVariantOf: Product }
 
+function sanitizeProduct(product: Product): Product {
+  return {
+    ...product,
+    description: product.description
+      ? sanitizeHtml(product.description)
+      : product.description,
```

Comment:
Which `product.description` values should not be sanitized but escaped?

Reply:
What do you mean?

Reply:
Because it can possibly have different falsy states, such as `null` or `undefined`. I don't want to change the default falsy value returned by the headless API, so I return whatever it returns me in these case.